build.gradle

```diff
@@ -140,9 +132,16 @@ allprojects {
     }
 
     // Distribution
-    task createDistribution(type: Sync) {
-        // source directory depends on the subproject
-        into distDir
+    if(project.name.contains('shaded')) {
+        task createDistribution(type: Sync, dependsOn: 'shadowJar') {
```

Comment:
It does seem like this should be in `fdb-record-layer-shaded.gradle`. Something like: `createDistribution.dependsOn { shadowJar }`.

Reply:
made this change, appears to work as desired. thanks.